tests/test.py

```diff
@@ -296,103 +301,101 @@ def test_015():
         assert out == "0"
-    
+
     kube_delete_chi("test-016-dict")
 
 @TestScenario
 @Name("test-017-multi-version. Test certain functions across multiple versions")
 def test_017():
     create_and_check("configs/test-017-multi-version.yaml", {"pod_count": 6, "do_not_delete": 1})
     chi = "test-017-multi-version"
-            
+
     test_query = "select 1 /* comment */ settings log_queries=1"
-    for shard in [0,1,2,3,4,5]:
+    for shard in range(6):
         host = f"chi-{chi}-default-{shard}-0"
-        clickhouse_query(chi, host = host, query = test_query)
-        out = clickhouse_query(chi, host = host, query = "select query from system.query_log order by event_time desc limit 1")
-        ver = clickhouse_query(chi, host = host, query = "select version()")
+        clickhouse_query(chi, host=host, query=test_query)
+        out = clickhouse_query(chi, host=host,
+                               query="select query from system.query_log order by event_time desc limit 1")
+        ver = clickhouse_query(chi, host=host, query="select version()")
 
         print(f"version: {ver}")
         print(f"queried: {test_query}")
         print(f"logged: {out}")
 
     kube_delete_chi(chi)
 
-
 # End of test scenarios
 
-kubectl.namespace="test"
-# operator_version="latest"
-operator_version="0.9.2"
-
-clickhouse_template="templates/tpl-clickhouse-stable.yaml"
-# clickhouse_template="templates/tpl-clickhouse-20.1.4.14.yaml"
-
-clickhouse_version=get_ch_version(clickhouse_template)
-
 if main():
-    with Module("main"):
-        with Given(f"ClickHouse template {clickhouse_template}"):
-            with And(f"ClickHouse version {clickhouse_version}"):
+    with Module("main", flags=TE):
+        with Given(f"ClickHouse template {settings.clickhouse_template}"):
+            with And(f"ClickHouse version {settings.clickhouse_version}"):
                 1 == 1
 
         with Given("clickhouse-operator is installed"):
-            # assert kube_get_count("pod", ns = "kube-system", label = "-l app=clickhouse-operator")>0, error()
-            with And(f"Set operator version {operator_version}"):
-                set_operator_version(operator_version)
-        
-        with Given(f"Clean namespace {kubectl.namespace}"): 
-            kube_deletens(kubectl.namespace)
-            with And(f"Create namespace {kubectl.namespace}"):
-                kube_createns(kubectl.namespace)
-
-    #    with Module("examples", flags=TE):
-    #        examples=[test_examples01_1, test_examples01_2, test_examples02_1, test_examples02_2]
-    #        for t in examples:
-    #            run(test=t, flags=TE)
-        
+            if kube_get_count("pod", ns='--all-namespaces', label="-l app=clickhouse-operator") == 0:
+                installer = get_full_path('../deploy/operator-web-installer/clickhouse-operator-install.sh')
+                installer = shell(f"bash -x {installer}")
```

Comment:
Why do you need a shell script? You can use kubectl apply instead

Reply:
fixed